src/Discovery/Discovery.tsx

```diff
@@ -367,21 +367,17 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
     columns.push({
       title: <div className='discovery-table-header'>Data Availability</div>,
       filters: [{
-        text: <React.Fragment><UnlockOutlined />&nbsp;Accessible</React.Fragment>,
+        text: <React.Fragment><UnlockOutlined />&nbsp;Available</React.Fragment>,
         value: AccessLevel.ACCESSIBLE,
         id: 'accessible-data-filter',
       }, {
-        text: <React.Fragment><LockFilled />&nbsp;Unaccessible</React.Fragment>,
-        value: AccessLevel.UNACCESSIBLE,
-        id: 'unaccessible-data-filter',
+        text: <React.Fragment><DashOutlined />&nbsp;Not Available</React.Fragment>,
+        value: AccessLevel.NOT_AVAILABLE,
```

Comment:
To double check, you do have `AccessLevel.UNACCESSIBLE` for data, but you don't want to have a filter for it in the table? How would those data show up in table?

Reply:
That's right. Those data will show up in the table with no row value in the Data Availability column. We're de-emphasizing the representation of locked / unaccessible data in the UI.

Reply:
I have this branch deployed in qa-heal ~ https://qa-heal.planx-pla.net/

Reply:
I personally think this is a interesting UX, and probably is more HEAL specific so likely it will be needed to be more customizable

Can you explicitly mention it (removing `AccessLevel.UNACCESSIBLE` from table sorting options) in `Improvements` section?